enrolments/tests/test_validators.py

```diff
@@ -52,6 +53,73 @@ def test_validate_classes_in_session(self):
             session_other,
         )
 
+    def test_validate_enrolment_in_session(self):
+        session1 = Session.objects.create(season=Session.SPRING, year="2020")
+        session2 = Session.objects.create(season=Session.FALL, year="2019")
+        class1_in_session1 = Class.objects.create(
+            name="Class 1 in Session 1",
+            session=session1,
+            facilitator=self.user,
+            attendance={},
+        )
+        class2_in_session1 = Class.objects.create(
+            name="Class 2 in Session 1",
+            session=session1,
+            facilitator=self.user,
+            attendance={},
+        )
+        class1_in_session2 = Class.objects.create(
+            name="Class 1 in Session 2",
+            session=session2,
+            facilitator=self.user,
+            attendance={},
+        )
```

Comment:
could use 'self.class_in_session' in the case where you replace `session2` with the session created in setUp()